### PR TITLE
Release @google-cloud/firestore v0.20.0

### PR DESCRIPTION
**Pull request by JustinBeckwith**
This pull request was generated using releasetool.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,45 @@
 
 [1]: https://www.npmjs.com/package/@google-cloud/firestore?activeTab=versions
 
+## v0.20.0
+
+01-16-2019 13:14 PST
+
+### Bug Fixes
+- refactor: change timestampsInSnapshots default to true.. ([#520](https://github.com/googleapis/nodejs-firestore/pull/520))
```
Comment (mikelehen) [inline CHANGELOG.md]: FYI- This is actually a breaking change (we're changing the default behavior for people not using timestampsInSnapshots). Should we flag it as more than just "refactor" somehow?  (sorry I was probably supposed to name my PR differently)

Comment (JustinBeckwith) [reply CHANGELOG.md]: Yeah - we should totally do that.  I'm not super up to date on the change.  Can you add that to the changelog at the top?  See this for how I flag breaking changes:
https://github.com/googleapis/google-auth-library-nodejs/pull/594/files

Comment (mikelehen) [reply CHANGELOG.md]: Thanks.  I added a breaking change notice for timestampsInSnapshots, and a deprecation notice for v1beta1 gapic client.  Let me know if you want any tweaks.  And if there's anything more we should do in order to deprecate v1beta1, let me know.

**Comment by mikelehen** [comment]
I consent to my changes being included so I've added the cla:yes label. :-)

**Pull request merged**